RemixedDungeon/src/main/java/com/watabou/pixeldungeon/RemixedDungeon.java

```diff
@@ -48,15 +49,19 @@
 import com.watabou.pixeldungeon.utils.Utils;
 import com.watabou.pixeldungeon.windows.elements.Tool;
 
+import java.util.ArrayList;
+import java.util.List;
 import java.util.Locale;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
+import com.nyrds.pixeldungeon.modding.Event;
 
 import javax.microedition.khronos.opengles.GL10;
 
 public class RemixedDungeon extends Game {
 
 	public static final double[] MOVE_TIMEOUTS = new double[]{250, 500, 1000, 2000, 5000, 10000, 30000, 60000, Double.POSITIVE_INFINITY };
+	public static List<Event> events = new ArrayList<Event>();
```

Comment:
Map<String, List<Event>>?

Reply:
Why second parameter is List?May be Map<String, Void> where first param is name of event and second is callback?

Reply:
Yes, in next commit